main.go

```diff
@@ -49,13 +50,21 @@ func init() {
 func main() {
 	flag.Parse()
 
+	if *routePrefix == "/" {
+		*routePrefix = ""
+	}
+	if *routePrefix != "" && (*routePrefix)[0:1] != "/" {
+		log.Fatalf("-web.route-prefix value must begin with `/` (was `%s`)", *routePrefix)
+	}
```

Comment:
This requirement is inconsistent with the Prometheus server, which silently prepends the `/`. I think we can do the same here. Suggestion:

```Go
if *routePrefix != "" {
	*routePrefix = "/" + strings.Trim(*routePrefix, "/")
}
```

Reply:
Ok, seems reasonable. I'll try that out.